lms/static/sass/views/_login-register.scss

```diff
@@ -350,6 +350,8 @@ $sm-btn-linkedin: #0077b5;
         @extend %btn-secondary-grey-outline;
         width: 130px;
         padding: 0 0 0 ($baseline*2);
+        margin-right: 6px;
+        margin-bottom: $baseline;
         height: 34px;
```

Comment:
Gotcha. Apologies. Yeah, we can definitely table this and make it part of edX's clean up.